PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/sdk/deviceData/managers/UserConsentManager.java

```diff
@@ -113,7 +113,15 @@ private void updateConsentValue(
 
         switch (key) {
             case GDPR_PREBID_SUBJECT:
-                gdprPrebidSubject = preferences.getString(GDPR_PREBID_SUBJECT, null);
+                try {
+                    gdprPrebidSubject = preferences.getString(GDPR_PREBID_SUBJECT, null);
+                } catch (ClassCastException classCastException) {
```

Comment:
Need to polish the code here:
- Use curly brackets for if-else 
- Add a comment to the line where we try to extract the boolean value. Mention the bug in the comment
- wrap getBoolean into try-catch as well. Who knows what we can meet there. 
- keep the consistent formatting for `preferences.edit()` line - put each function on the separate row